### PR TITLE
chore: Sync Cargo.toml to Cargo.lock

### PR DESCRIPTION
**Pull request by epage**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -69,3 +69,15 @@ jobs:
...
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Install Rust
+      uses: dtolnay/rust-toolchain@stable
+      with:
+        toolchain: stable
+    - uses: Swatinem/rust-cache@v2
+    - name: "Is lockfile updated?"
+      run: cargo update --workspace --locked
```
Comment (eopb) [inline .github/workflows/rust.yml:83]: I'd never seen this trick before. I'll copy that for other projects :raised_hands:

**Review by eopb — APPROVED** [review]
Thank you :heavy_check_mark:

File: .github/workflows/rust.yml (hunk 1 above)
Comment (epage) [reply .github/workflows/rust.yml:83]: If you are curious, the settings for all of my projects is at https://github.com/epage/_rust

I have that as a merge-base for easy updating

Comment (eopb) [reply .github/workflows/rust.yml:83]: Thanks, that's awesome :eyes:

**Pull request merged**